galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -52,11 +52,6 @@ class ViewNamespaceSerializerContextMixin:
     def get_serializer_context(self):
         """Inserts distribution path to a serializer context."""
 
-        self._deprecation
```

Comment:
@fao89 looks like this call is needed only when we have no collections, if my db is empty and I try to call `/collections` I will get an error here. After a sync the error does not occur, as this field is populated on the filter I think that happens because empty queryset does not invoke the filters.